nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/ui/canvas/items/process-group/edit-process-group/edit-process-group.component.ts

```diff
@@ -71,6 +67,12 @@ export class EditProcessGroup extends TabbedDialog {
                     value: parameterContext.id,
                     description: parameterContext.component.description
                 });
+            } else {
+                this.parameterContextsOptions.push({
+                    text: parameterContext.id,
+                    value: parameterContext.id,
+                    disabled: true
+                });
```

Comment:
If the user doesn't have read access to the currently set parameter context we should not disable it as a selectable option in the dropdown. If they accidentally change the context in the dropdown, they have no way of setting it back to what it was before without cancelling the dialog and potentially losing other changes made in the dialog.